std-bits/base/src/main/java/org/enso/base/statistics/CorrelationStatistics.java

```diff
@@ -22,6 +22,30 @@ private void append(Double x, Double y) {
     totalXY += x * y;
   }
 
+  public long getCount() {
+    return count;
+  }
+
+  public double getTotalX() {
+    return totalX;
+  }
+
+  public double getTotalY() {
+    return totalY;
+  }
+
+  public double getTotalXX() {
```

Comment:
This is rather trivial, but a short comment saying what this statistic is could be helpful.

Like `Sum of x[i]*x[i]` or something like that.